tedana/workflows/tedana.py

```diff
@@ -131,11 +131,12 @@ def _get_parser():
                                 'delimited list'),
                           choices=['t1c', 'gsr'],
                           default=None)
-    optional.add_argument('--tedpca',
-                          dest='tedpca',
-                          help='Method with which to select components in TEDPCA',
-                          choices=['mle', 'kundu', 'kundu-stabilize'],
-                          default='mle')
+    optional.add_argument(
+        '--tedpca',
+        dest='tedpca',
+        help='Method with which to select components in TEDPCA',
```

Comment:
A shortened version of the information added to the `tedpca` function docstring about the options can also be added here. The default here should also be set to mdl to match the default in the function docstring.